css/view.css

```diff
@@ -3,6 +3,15 @@
   display: none !important;
 }
 
-span.tvm__text:not(:has(span)) {
+/* hides "see how you compare to x applicants" option next to reactivate premium button */
+
+.job-details-jobs-unified-top-card__job-insight--highlight:has(div):has(
+    div
+  ):has(div):has(svg[data-test-icon="lightbulb-medium"]) {
```

Comment:
The "see how your application compares to x applicants" section has a lightbulb icon to the left-hand side. None of the other list items do, so this code looks for a parent element that contains a third child `div`'s, and the final child `div` contains a the lightbulb `svg`.